data/src/test/java/org/apache/iceberg/data/DeleteReadTests.java

```diff
@@ -297,11 +322,14 @@ public void testEqualityDeletesSpanningMultipleDataFiles() throws IOException {
             table, Files.localOutput(temp.newFile()), Row.of(0), dataDeletes, deleteRowSchema);
 
     table.newRowDelta().addDeletes(eqDeletes).commit();
+    // At ths point, the table has (7 - 3) + (8 - 4) = 8 rows. 7 rows in all are deleted.
```

Comment:
Nit: the phrasing of this comment is a little hard for me to follow without reading it twice.

Maybe something like `// At ths point, the table has two data files. They have 7 rows and 8 rows with 3 deleted rows and 4 deleted rows respectively.`

But again this is just a test so I wouldn’t block on this.